view.php

```diff
@@ -133,9 +133,19 @@
 $turnitintooltwoassignment = new turnitintooltwo_assignment($turnitintooltwo->id, $turnitintooltwo);
 
 // Migration tool
-if ($migrated === 1) {
+$notice = array();
+if ($migrated == 1 || $migrated == 2) {
```

Comment:
Should these use `===` (identical - strict comparison) instead of `==` (equals - loose comparison) ?

Reply:
I'll change them.